compiler/modules/comm.diagnostic/printer.bal

```diff
@@ -0,0 +1,88 @@
+import ballerina/io;
+import ballerina/file;
+
+public type Error error<Diagnostic>;
+
+public type Printer object {
+    public function print(Error diagnostic);
+    public function close() returns (file:Error|io:Error)?;
+};
+
+public class StdErrorPrinter {
```

Comment:
`Std` plus `Error` isn't a good combo.  Plus `StdError` and `Html` isn't a good combo.

I suggest StdErrTextPrinter.  Or mayber better to specify io:stderr|io:stdout in a constructor. Then you could call it ConsolePrinter.

Reply:
add option `redirectErrors` that can be used to send error longs to std:out